packages/central-server/__tests__/hl7fhir/materialised/MedicationRequest.test.js

```diff
@@ -0,0 +1,253 @@
+import { formatRFC7231 } from 'date-fns';
+import { fake, fakeReferenceData } from '@tamanu/fake-data/fake';
+import { createTestContext } from '../../utilities';
+import { ADMINISTRATION_FREQUENCIES, DRUG_ROUTES, DRUG_UNITS } from '@tamanu/constants';
+import { formatFhirDate } from '@tamanu/shared/utils/fhir/datetime';
+
+const INTEGRATION_ROUTE = 'fhir/mat';
+
+describe(`Materialised FHIR - MedicationRequest`, () => {
+  let ctx;
+  let app;
+  let resources;
+
+  beforeAll(async () => {
+    ctx = await createTestContext();
+    app = await ctx.baseApp.asRole('practitioner');
+
+    const {
+      Facility,
+      Location,
+      Department,
+      Patient,
+      ReferenceData,
+      User,
+      Encounter,
+      FhirPatient,
+      FhirOrganization,
+      FhirPractitioner,
+      FhirEncounter,
+    } = ctx.store.models;
+
+    const [practitioner, patient, drug1, drug2, facility] = await Promise.all([
+      User.create(fake(User)),
+      Patient.create(fake(Patient)),
+      ReferenceData.create({ ...fakeReferenceData(), type: 'drug' }),
+      ReferenceData.create({ ...fakeReferenceData(), type: 'drug' }),
```

Comment:
Looks like u can do this!
```suggestion
      ReferenceData.create(fakeReferenceData('drug')),
      ReferenceData.create(fakeReferenceData('drug')),
```